_episodes/43.md

```diff
@@ -0,0 +1,200 @@
+
+### What is Alluxio?
+
+Alluxio is world’s first open source data orchestration technology for analytics
+and AI for the cloud. It provides a common interface enabling computation
+frameworks to connect to numerous storage systems through a common interface.
+Alluxio’s memory-first tiered architecture enables data access at speeds orders
+of magnitude faster than existing solutions. Alluxio was originally developed at
+Berkley Amp Labs, [formerly known as Tachyon](
+https://amplab.cs.berkeley.edu/wp-content/uploads/2014/11/2014_socc_tachyon.pdf),
+and was less focused on caching and data orchestration and more focused on
+fault-tolerance via lineage and other techniques borrowed from Spark. 
+
+Alluxio lies between data driven applications, such as Trino and Apache Spark,
+and various persistent storage systems, such as Amazon S3, Google Cloud Storage,
+HDFS, Ceph, and Minio. Alluxio unifies the data stored in these different
+ storage systems, presenting unified client APIs and a global namespace to its
+upper layer data driven applications.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/43/alluxio-architecture.png"/>
+</p>
+
+Alluxio is commonly used as a distributed shared caching service so compute
+engines talking to Alluxio can transparently cache frequently accessed data,
+especially from remote locations, to provide in-memory I/O throughput. Alluxio
+also makes using Object storage much simpler as you will not hit the storage
+in the cloud if the file is already cached by Alluxio. Alluxio also handles the
+complex mapping client interface requests to underlying storage interfaces.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/43/inside-alluxio.png"/>
+</p>
+
+Source: <https://docs.alluxio.io/os/user/stable/en/Overview.html>
+
+### What is data orchestration?
+
+A data orchestration platform abstracts data access across storage systems,
+virtualizes all the data, and presents the data via standardized APIs with
+global namespace to data-driven applications. In the meantime, it should have
+caching functionality to enable fast access to warm data. In summary, a data
+orchestration platform provides data-driven applications Data Accessibility,
+Data Locality, and Data Elasticity.
+
+Source: <https://www.alluxio.io/blog/data-orchestration-the-missing-piece-in-the-data-world/>
+
+### Trino and Alluxio: Expedia use case
+
+Expedia needed to have the ability to query cross cluster over different regions
+while simplifying the interface to their local data sources.
+
+<p align="center">
+<img align="center" width="100%" height="100%" src="/assets/episode/43/expedia-trino-alluxio.png"/>
+</p>
+
+Source: [Unifying Cross-region Access in the Cloud at Expedia Group — The Path Toward Data Mesh in the Brand World](https://www.alluxio.io/blog/unifying-cross-region-access-in-the-cloud-at-expedia-group-the-path-toward-data-mesh-in-the-brand-world/)
```

Comment:
I know its a source.. but get rid of the title case anyway .. its not readable here